### DISCUSSION TITLE
Discussion about programming crazy

### DISCUSSION THREAD

**Discussion by baphomate** [baphomate/baphomate#3] [Crazy developers]
be a crazy developer wkwkwk

**Comment by Arynz-C** [comment]
Anjir🗿

**Reply by baphomate** [reply]
Siapa aja yang join bre

**Reply by Arynz-C** [reply]
Kaya nya gw sndiri

**Comment by GtGnzz** [comment]
Eeee... Hai

**Reply by baphomate** [reply]
Bikin roket pake python hayuk awowowk

**Reply by GtGnzz** [reply]
> hayuk

WKwkwk gas ae

**Comment by Michaelbai2631** [comment]
Hello

**Comment by baphomate** [comment]
Hallo semua mari kita diskusi bikin bot ai yang sange berat awowowowk

**Reply by Arynz-C** [reply]
Bjird

**Reply by baphomate** [reply]
Ide nya si @GtGnzz

**Reply by Arynz-C** [reply]
Sesad

**Reply by baphomate** [reply]
Iya bjir gak nongol dia kan dia otak dari semua nya wkwkwk @GtGnzz cevet on ngap kita bikin ai sangean

**Comment by Arynz-C** [comment]
Sekalian bisa di ewe🗿

**Reply by baphomate** [reply]
Nah itu renca bikin bot ai nya bersifat pisik, kalo masalah tubuh pake mayat aja

**Reply by GtGnzz** [reply]
> Nah itu renca bikin bot ai nya bersifat pisik, kalo masalah tubuh pake mayat aja

Anjr bikin kepala aja gaush pke tubuh😅

**Reply by baphomate** [reply]
Bjirr gak enak bre masa di spong doang

**Reply by baphomate** [reply]
We lagi bikin project we fish nih review nya ada di grup video nya

**Comment by baphomate** [comment]
Situs baphonate udah normal bre 

[Forum baphonate inc](https://forums.baphomate.rf.gd)
